primeNbr.js

```diff
@@ -0,0 +1,14 @@
+const isPrime = (number) => {
+    if (number < 1 ) return false
+
+    let isPrimeNumber = true
+    for (let index = 2; index<number; index++){
+        if (number % index === 0){
+            isPrimeNumber = false
+            break
+        }
+    }
+    return isPrimeNumber
+}
+const u = isPrime(7)
+console.log(u)
```

Comment:
Variable names must be more descriptive, or if they are for testing, you can remove them after testing